maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/MavenUtil.java

```diff
@@ -486,6 +498,29 @@ public static boolean isPomFile( IFile pomFile )
             pomFile.getParent() instanceof IProject;
     }
 
+    public static void importProject( String location, IProgressMonitor monitor )
```

Comment:
How does the "new liferay plugin project" that creates a maven project work?  How does it "import" the newly created project?  Can any code there be shared with what is below?